### DISCUSSION TITLE
Synced Switches

### DISCUSSION THREAD

**Discussion by mnedix** [crystian/ComfyUI-Crystools#116] [Ideas]
I love the switch node and I use it quite often in my workflows. I was thinking that perhaps syncing switches could also be valuable. A use-case could be:
1. the workflow has multiple input checkpoints, e.g: Flux1.D with external CLIP/AE, .GGUF, Flux/XL with baked Clip/AE etc. 
2. you can use a Switch Any to easily switch between 2 of them
3. at the output, the Image Saver gets the modelname via a Widget to String node (from KJNodes)
4. however, every time you change the input you also have to change from which Widget the Image Saver gets the modelname 
(I hope the above makes sense).
So it would be great if we had a way to sync the Switches so when the input switch is True, the one before the Image Saver is also True 

Best,

**Comment by crystian** [comment]
Um, I'm not sure about the request. I think you can have two saved pictures with different names/folders at the end of each branch. Does that make sense?
